.github/workflows/check-dist.yml

```diff
@@ -21,10 +21,10 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
 
       - name: Set Node.js 16.x
-        uses: actions/setup-node@v1
+        uses: actions/setup-node@v3
```

Comment:
I believe we can use npm cache for setup-node action to increase performance

Reply:
Done. Thank you.